README.md

```diff
@@ -69,11 +69,20 @@ Example:
 ```js
 const fs = require('fs-extra')
 
+// Async with promises:
+fs.copy('/tmp/myfile', '/tmp/mynewfile')
+  .then(() => console.log("success!"))
```

Comment:
Probably not that important, but for the purpose of consistency in using `'`, wouldn't it be better `console.log("success!")` replaced with `console.log('success!')` since we use `'` pretty much everywhere else? It actually has been there since the beginning (I guess). I just saw it now and thought that's be nice that we fix it here :smile: